apps/meteor/app/slackbridge/server/RocketAdapter.js

```diff
@@ -228,11 +228,11 @@ export default class RocketAdapter {
 	}
 
 	async getChannel(slackMessage) {
-		return slackMessage.channel ? this.findChannel(slackMessage.channel) || this.addChannel(slackMessage.channel) : null;
+		return slackMessage.channel ? (await this.findChannel(slackMessage.channel)) || this.addChannel(slackMessage.channel) : null;
 	}
 
 	async getUser(slackUser) {
-		return slackUser ? this.findUser(slackUser) || this.addUser(slackUser) : null;
+		return slackUser ? (await this.findUser(slackUser)) || this.addUser(slackUser) : null;
```

Comment:
notice that in other calls to `this.addUser` they put an `await` first, should we add one too?

Reply:
No, in this case where we don't add the `await` is because we are returning the promise directly, otherwise, it'll be a redundant return await.